src/core/server/http/types.ts

```diff
@@ -276,8 +285,33 @@ export interface InternalHttpServiceSetup
 
 /** @public */
 export interface HttpServiceStart {
-  /** Indicates if http server is listening on a given port */
-  isListening: (port: number) => boolean;
+  /**
+   * Access or manipulate the Kibana base path
+   * See {@link IBasePath}.
+   */
+  basePath: IBasePath;
+
+  /**
+   * Auth status.
+   * See {@link HttpAuth}
+   */
+  auth: HttpAuth;
```

Comment:
Should some of these APIs be now flagged as `deprecated` on the setup contract?

Reply:
yes, it cannot be called in `setup` anyway.